src/components/text-input.components.js

```diff
@@ -0,0 +1,121 @@
+import $ from 'jquery';
+
+export default class TextInput {
+    
+    constructor(stringQuery, textEnable = false, required = true, textValid = true){
+
+        this.stringQuery = stringQuery;
+
+        this.textInput = $(`${stringQuery} input[type="text"]`); 
+
+        this.emailInput = $(`${stringQuery} input[type="email"]`);
+        
+        this.textValid = textValid;
+        this.textEnable = textEnable;
+        this.required = required;
+    }
+
+    getTextInput(){
+        return this.textInput.val();
+    }
+
+    setTextInput(value){
+        this.textInput = this.textInput.val(value);
+    }
+
+    getTextValid(){
+        return this.textValid;
+    }
+
+    setTextValid (boolean){
+        this.textValid = boolean;
+    }
+
+    isTextValid(){
```

Comment:
Why you are return the value instead of a boolean value? isTextValid should return a boolean.

Reply:
I fixed this isTextValid return a text, that's not ''.